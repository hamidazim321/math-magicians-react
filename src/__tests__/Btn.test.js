import { render } from '@testing-library/react';
import CalcBtn from '../components/Btn';

test('should render', () => {
  const { asFragment } = render(<CalcBtn text="hello" handleClick={() => 'hello'} />);
  expect(asFragment()).toMatchSnapshot();
});
