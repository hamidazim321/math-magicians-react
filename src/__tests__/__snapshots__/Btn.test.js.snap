// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<DocumentFragment>
  <button
    class="btn "
    type="button"
  >
    hello
  </button>
</DocumentFragment>
`;
